### ISSUE TITLE
Seems to be broken after slider revolution update 

### ISSUE THREAD

**Issue by ehausen** [Nikschavan/revslider-search-replace#6]
I'm now on Slider Revolution 6.0.9 and seem to be getting error. The slider seems to have gone through a major update

**Comment by ethanclevenger91** [comment]
Yeah, the RevSliderSlider::replaceImageUrlsFromData function has been removed

**Comment by ethanclevenger91** [comment]
They moved their search/replace functionality into an add-on: https://codecanyon.net/item/slider-revolution-responsive-wordpress-plugin/2751380/comments?page=1016&filter=all#comment_22926455

Probably much less reliable to integrate with.

**Comment by Nikschavan** [comment]
Does installing the add-on bring this back as working?

I have not been using Revolution slider for some time now so I cannot test this myself.

**Comment by khanakia** [comment]
You can use this repo it supports Slider Revolution 6.0 or greater
https://github.com/khanakiaphp/revslider-search-replace
